report/4.Design/Robot_Design.tex

```diff
@@ -1,51 +1,50 @@
-There where a multiple problems with this early design, first the top part of the turret where not stable enough for the robot to achieve the \textit{robustness}.
+The turret has been through different iterations during the development. 
+On \autoref{fig:TurretEarly}, the first iteration of the turret can be seen.
+There were multiple problems with this early design, first of all, the top part of the turret was not stable enough for the robot to achieve the wanted \textit{robustness}.
 \figur{0.5}{images/TurretEarly}{The figure shows the first iteration of the turret design}{fig:TurretEarly}
 
-After the first iteration an second attempt where made in order to correct the issues of the first. 
-On \autoref{fig:TurretMiddle} first the top part of the turret turned out to be too heavy for the motor to handle. 
+After the first iteration, a second attempt was made in order to address these issues. 
+On \autoref{fig:TurretMiddle}, the top part of the turret turned out to be too heavy for the motor to handle. 
 \figur{0.5}{images/TurretMiddle}{The figure shows the second iteration of the turret design}{fig:TurretMiddle}
 
-
-
-Hence the next iteration of the turret where designed to be lighter. 
-As can be seen on \autoref{fig:TurretFinal} the top of the turret has been significantly reduced in both complexity and weight. 
-The most significant reduction on the turret the new laser mounted on the existing webcam which has been moved closer to the rotating axis hence reducing the power needed to move the turret. 
+To solve this, the next iteration introduced a lighter model, as seen on \autoref{fig:TurretFinal}.
+The top of the turret has been significantly reduced in both complexity and weight. 
+The most significant reduction on the turret being the new laser, that is mounted on the webcamera.
+Likewise, the camera has been moved closer to the rotating axis, resulting in a reduction of the power needed to move the turret. 
 \figur{0.4}{images/TurretFinal}{The figure shows the final iteration of the turret design}{fig:TurretFinal}
 
 \subsection{Cable management}
-In order to make the robot according to the criteria of \textit{correctness} and \textit{robustness} the cables attached to the different components of the robot should have good cable management. 
+In order to make the robot comply to the criteria of \textit{correctness} and \textit{robustness}, the cables attached to the different components of the robot should have good cable management. 
 The cable management should make the robot able to move without unplugging any cable.
-However since the robot is constructed primarily using LEGO Technic, the cables are robust and hard to manage hence only the cables from the webcam and laser is manageable. 
\ No newline at end of file
+However, since the robot is constructed primarily using LEGO Technic, the cables are robust and hard to manage, resulting in only the cables from the webcam and laser being manageable. 
```

Comment:
```suggestion
However, since the robot is constructed primarily using LEGO Technic, the cables are robust and difficult to manage, resulting in only the cables from the webcam and laser being manageable. 
```